oauthproxy.go

```diff
@@ -736,14 +736,14 @@ func (p *OAuthProxy) OAuthStart(rw http.ResponseWriter, req *http.Request) {
 	prepareNoCache(rw)
 	nonce, err := encryption.Nonce()
 	if err != nil {
-		logger.Printf("Error obtaining nonce: %v", err)
+		logger.Errorf("Error obtaining nonce: %s", err.Error())
```

Comment:
Why change from `%v` and `err` to `%s` and `err.Error()`, is the former pairing not the more idiomatic style in Go?

Reply:
Very much so. I don't remember making this change, though I only edited the function name. XD
Not sure where this came in, but will change back.